seacrowd/sea_datasets/asr_malcsc/asr_malcsc.py

```diff
@@ -0,0 +1,191 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# no bibtex citation
+_CITATION = ""
+_DATASETNAME = "asr_malcsc"
+_DESCRIPTION = """\
+This open-source dataset consists of 5 hours of transcribed Malay
+conversational speech on certain topics, where ten conversations between five
+pairs of speakers were contained.
+"""
+
+_HOMEPAGE = "https://magichub.com/datasets/malay-conversational-speech-corpus/"
+_LANGUAGES = ["zlm"]
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value
+_LOCAL = False
+_URLS = {
+    _DATASETNAME: "https://magichub.com/df/df.php?file_name=Malay_Conversational_Speech_Corpus.zip",
+}
+_SUPPORTED_TASKS = [Tasks.TEXT_TO_SPEECH, Tasks.SPEECH_RECOGNITION]
```

Comment:
```suggestion
_SUPPORTED_TASKS = [Tasks.SPEECH_RECOGNITION]
```
This dataset is for transcription so you might've been thinking of `Tasks.SPEECH_TO_TEXT`. But seeing other speech dataloaders, just `Tasks.SPEECH_RECOGNITION` seems sufficient.